Core/clim-core/commands/processor.lisp

```diff
@@ -38,9 +38,14 @@
         (when (and delimiter (delimiter-gesture-p delimiter))
           (read-gesture :stream stream))))
     (with-delimiter-gestures (*command-argument-delimiters* :override t)
-      (setq command-args (funcall (parser (gethash command-name
-                                                   *command-parser-table*))
-                                  stream)))
+      ;; The null context is a barrier which prevents other commands from
+      ;; being selected during reading arguments. -- jd 2020-09-03
+      (with-input-context ('null :override t) ()
+          (setq command-args (funcall (parser (gethash command-name
+                                                       *command-parser-table*))
+                                      stream))
+        ;; Impossible clause.
+        (t (error "U Can't Touch This"))))
```

Comment:
Why is this needed? If it is actually reachable (or in any case, really), the error message should provide information and explain the situation.

Reply:
It is not reachable, if it is reached then we have a bug in input context processing. it is similar to "You lose" errors in other parts of the code.

Reply:
Maybe "An error an error in McCLIM caused code to be executed that should never be reached. Please report this as a bug."?

Reply:
removed the clause whatsoever.